pkg/platformconfig/types.go

```diff
@@ -100,10 +99,12 @@ type Kubernetes struct {
 
 // default values for created ingresses
 type IngressConfig struct {
-	EnableSSLRedirect bool   `json:"enableSSLRedirect,omitempty"`
-	TLSSecret         string `json:"tlsSecret,omitempty"`
-	IguazioAuthURL    string `json:"iguazioAuthURL,omitempty"`
-	IguazioSignInURL  string `json:"iguazioSignInURL,omitempty"`
+	EnableSSLRedirect          bool     `json:"enableSSLRedirect,omitempty"`
+	TLSSecret                  string   `json:"tlsSecret,omitempty"`
+	IguazioAuthURL             string   `json:"iguazioAuthURL,omitempty"`
+	IguazioSignInURL           string   `json:"iguazioSignInURL,omitempty"`
+	AllowedAuthenticationModes []string `json:"allowedAuthenticationModes,omitempty"`
```

Comment:
`[]string` -> `[]ingress.AuthenticationMode` ?

Reply:
At the beginning I started with that, but that brings import loops hell:
the `ingress` package imports the `platformconfig` package, and your suggested change will make the `platformconfig` package import the `ingress` package.
Also this change make several other packages (like `pkg/dashboard/resource` because of `frontendspec`) to have to import `ingress` which introduce several others import loops